### ISSUE TITLE
Unit tests should not need the network

### ISSUE THREAD

**Issue by darius** [aimacode/aima-python#490]
`test_nlp.py` has a call to `loadPageHTML` -- I found out about it when running `py.test` seemed to hang up for minutes. (It eventually finished. I was downloading the Anaconda installer at the same time.)

Tests should try to use only local data -- maybe from `aima-data`? Or keep it cached after the first run. I'd rate this as a nice-to-have, not a big problem, currently.

**Comment by MrDupin** [comment]
I worked on this a bit and I found that only `test_stripRawHTML` requires an internet connection. If we remove that test, everything will still work fine and there will be no need for network connectivity.

Shouldn't we though keep the test to make sure we don't accidentally break something in the used functions?

If we are to keep it, I tried some different wikipedia pages to make the process faster. If instead of "https://en.wikipedia.org/wiki/Ethics" we use "https://en.wikipedia.org/wiki/Terminus_post_quem" (which is significantly shorter) we reduce the testing time by about 1 second on bad connections. I don't know if that one second is worth it, but we could try.
